### PR TITLE
do not block time entry custom fields

### PR DESCRIPTION
**Pull request by machisuji**
https://community.openproject.org/work_packages/22116/activity

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://fix-time-entry-permitted-params.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://fix-time-entry-permitted-params.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

File: app/models/permitted_params.rb [hunk 1]
```diff
@@ -303,8 +303,9 @@ def project(instance = nil)
   end
 
   def time_entry
-    params.fetch(:time_entry, {}).permit(:hours, :comments, :work_package_id,
-                                       :activity_id, :spent_on, custom_field_values: [])
+    params.fetch(:time_entry, {}).permit(
+      :hours, :comments, :work_package_id, :activity_id, :spent_on,
+      custom_field_values: custom_field_keys)
```
Comment (ulferts) [inline app/models/permitted_params.rb]: Why are you not using what is already in place for that for other customized objects, e.g

https://github.com/opf/openproject/blob/fix/time_entry_permitted_params/app/models/permitted_params.rb#L173

Comment (machisuji) [inline app/models/permitted_params.rb]: As I said the question is if we actually want to whitelist or not. This is not whitelisting but accepting anything at all for custom field values. But seeing as this is already done it's ok then I guess.

Comment (machisuji) [inline app/models/permitted_params.rb]: @ulferts Updated it to use `#custom_field_values`.

**Pull request merged**